test/unit/schema/mssql.js

```diff
@@ -76,11 +76,11 @@ describe("MSSQL SchemaBuilder", function() {
 
   it('test drop primary', function() {
     tableSql = client.schemaBuilder().table('users', function() {
-      this.dropPrimary();
+      this.dropPrimary('testconstraintname');
     }).toSQL();
 
     equal(1, tableSql.length);
-    expect(tableSql[0].sql).to.equal('ALTER TABLE [users] DROP PRIMARY KEY');
+    expect(tableSql[0].sql).to.equal('ALTER TABLE [users] DROP CONSTRAINT [testconstraintname]');
```

Comment:
Yep, DROP CONSTRAINT only. See this article, section "Using Transact-SQL"
https://msdn.microsoft.com/en-us/library/ms190621.aspx